Products/CMFPlone/skins/plone_scripts/queryCatalog.py

```diff
@@ -7,8 +7,8 @@
 ##parameters=REQUEST=None,show_all=0,quote_logic=0,quote_logic_indexes=['SearchableText','Description','Title'],use_types_blacklist=False,show_inactive=False,use_navigation_root=False
 ##title=wraps the portal_catalog with a rules qualified query
 
+from plone.base.navigationroot import get_navigation_root
```

Comment:
@jensens The import in RestrictedPython is failing. Can we replace this script?

Reply:
If we finish #3515 all are gone.

Reply:
Lets try in https://github.com/plone/Products.CMFPlone/pull/3960
For grep the word is difficult.

Reply:
this is done and PR is green